runtime/src/bank.rs

```diff
@@ -5308,6 +5318,13 @@ impl Bank {
         }
     }
 
+    /// true if we should include the slot in account hash
+    fn include_slot_in_hash(&self) -> bool {
```

Comment:
callers of this function begin the application of the feature activation to the hash calculation.